documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md

```diff
@@ -0,0 +1,430 @@
+2. An evaluation facility is a legal entity or part of a legal entity that employs evaluators to conduct evaluations in accordance with these rules of procedure.
+3. The evaluation facility shall carry out its activities in a non-discriminatory and impartial manner in accordance with section 2.7 subsections 1 and 2 and shall have the necessary resources for the evaluation.
+4. The evaluation facility shall ensure the confidentiality of information on and arising from the evaluation procedures.
+5. The evaluation facility may use its own or external evaluators to carry out the evaluation.
+For external evaluators, the same competence requirements apply as in §2.4 for evaluators employed by the evaluation facility.
+6. The evaluation facility shall demonstrate and maintain adequate liability insurance.
+7. An evaluation facility may only carry out evaluations for the IDS if it has been recognized by the IDS certification body.
+
+### §2.4 Evaluators     
+1. Evaluators are natural persons who carry out evaluations for an evaluation facility in accordance with these rules of procedure.
+2. They must have sufficient competence to carry out the evaluations.
+This requires adequate training and experience.
+3. The evaluation requires a university degree or equivalent in the fields relevant to the evaluation.
+In particular, evaluators must have sufficient knowledge of the following subjects:
+  * a. IDS certification procedure;
+  * b. IDSA reference architecture;
+  * c. Technical requirements for IT security;
+  * d. ISO/IEC 270xy standards (or equivalent);
+  * e. ISO/IEC 15408 standard (or equivalent);
+  * f. Cryptography;
+  * g. Vulnerability analysis and penetration tests;
+  * h. Tool knowledge (in particular development tools, analysis, and attack tools).
+
+    *Requirement (d) is relevant only within the scope of the operational environment certification.*
+
+    *Requirements (e) to (h) are only relevant within the scope of component certification.*      
+
+4. Evaluators must have at least four years (full-time equivalent) experience in the field of information technology and at least two years (full-time equivalent) experience in the field of testing IT products or information security. The activity as IT security officer or consultant in the aforementioned areas is not an activity within the meaning of sentence 1.
+
+5. Evaluators must also have the necessary personal ability and reliability to carry out the evaluation.
+These include:
+  * a. Ability to deal with conflicts;
+  * b. Ability to work as part of a team;
+  * c. Knowledge of the English language;
+  * d. Basic knowledge of project management    
+6. Evaluators must be independent within the meaning of § 2.7 (3) and perform their duties in a non-discriminatory and impartial manner. They shall not evaluate an organization or component if there is a risk of a conflict of interest within the meaning of Art. 2.7 paragraph 4.
+
+### §2.5 Accreditation      
```

Comment:
```suggestion
### §2.5 Approval      
```

Reply:
Accreditation is only the correct term for the procedure described in the annex.

Reply:
accepted and adjusted